htdocs/sass/rcloud-discover.scss

```diff
@@ -99,6 +99,10 @@ progress::-webkit-progress-value {
 .grid-item h1{
     font-size: 17px;
     margin: 0;
+    /* firefox */
+    white-space: pre-wrap;
+    word-break: break-all;
+    /* chrome */
     word-break: break-word;
```

Comment:
Chrome is happy with it, but it's considered an invalid value for the `word-break` property by Firefox.